src/graphics/text.c

```diff
@@ -84,6 +86,40 @@ int text_get_width(const uint8_t *str, font_t font)
     return width;
 }
 
+void text_fit_to_width(char *str, font_t font, int requested_width, const char *end_replacement)
+{
+    char *orig_str = str;
+    const font_definition *def = font_definition_for(font);
+    int end_replacement_width = (end_replacement) ? text_get_width((const uint8_t *)end_replacement, font) : 0;
```

Comment:
Use `string_from_ascii()` instead of a cast.

Reply:
Will do!